### PR TITLE
BB2-3268 Web Deployment Python Upgrade to 3.11.9

### PR DESCRIPTION
**Pull request by noor1027**
**JIRA Ticket:**
[BB2-3268](https://jira.cms.gov/browse/BB2-3268)

### What Should Reviewers Watch For?
1. Ensure that all dependencies in requirements.txt or other package management files are compatible with Python 3.11.
2. Ensure that any changes related to type annotations or syntax updates introduced in Python 3.11 are correctly implemented.
3. Look for any missing or unclear instructions on setting up Python 3.11 locally or in deployment environments.

If you're reviewing this PR, please check for these things in particular:

### What Security Implications Does This PR Have?

Please indicate if this PR does any of the following:

* Adds any new software dependencies
* Modifies any security controls
* Adds new transmission or storage of data
* Any other changes that could possibly affect security?

- [ ] Yes, one or more of the above security implications apply. This PR must not be merged without the ISSO or team
  security engineer's approval.

### Any Migrations?

* [ ] Yes, there are migrations
    * [ ] The migrations should be run PRIOR to the code being deployed
    * [ ] The migrations should be run AFTER the code is deployed
    * [ ] There is a more complicated migration plan (downtime,
      etc) 
* [x] No migrations

### CODE REVIEW

File: roles/app_install/tasks/main.yml [hunk 1]
```diff
@@ -74,7 +74,9 @@
   become_user: "{{ remote_admin_account }}"
   become: yes
   shell: |
-    source {{ app_py_virtual_env }}/bin/activate && pip3.8 install -r requirements/requirements.txt --no-index --find-links ./vendor/
+    source {{ app_py_virtual_env }}/bin/activate
+    python3 -m pip install --upgrade pip setuptools wheel
```
Comment (loganbertram) [inline roles/app_install/tasks/main.yml:78]: Even as convention, I like this better. Splitting out commands unless there's a compelling reason to && them on one line.

File: vars/common.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ platinum_source: "amzn2"
 
 #Python Settings
 python_bin_dir: "/usr/bin/"
-python_ver: "{{ '3.8' if platinum_source == 'amzn2' else '3.6' }}"
+python_ver: "{{ '3' if platinum_source == 'amzn2' else '3.6' }}"
```
Comment (loganbertram) [inline vars/common.yml:43]: Curious about the 3.6 fallback here.

**Review by loganbertram — APPROVED** [review]

**Review by JFU-NAVA-PBC — APPROVED** [review]
LGTM

checked the new image pushed to ECR
visual examed the console output of previous jenkins build

May be it is not in this PR's scope, how about the local docker ? and bluebutton-web-server CI check jenkins?

File: vars/common.yml (hunk 2 above)
Comment (noor1027) [reply vars/common.yml:43]: fall back will default version in amzn2 and we are keeping as is , with amzn2023 we may not needed as by default comes with 3.11

File: roles/app_install/tasks/main.yml (hunk 1 above)
Comment (noor1027) [reply roles/app_install/tasks/main.yml:78]: i agree and its easy to  understand and debug.

**Comment by noor1027** [comment]
> May be it is not in this PR's scope, how about the local docker ? and bluebutton-web-server CI check jenkins?

Sure we can lookinto after this migration

**Pull request merged**